pom.xml

```diff
@@ -62,6 +62,7 @@
         <module>notification-service</module>
         <module>pattern-detection-library</module>
         <module>coverage</module>
+        <module>workflow-service-sdk</module>
```

Comment:
It would be great to add one more module for notification service sdk.

Reply:
Open new Jira task for this https://issues.redhat.com/browse/FLPATH-208